src/ErniAcademy.Events.ServiceBus/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static class ServiceCollectionExtensions
             return services;
         }
 
-        internal static IServiceCollection ConfigureOptions<TOptions>(this IServiceCollection services, string sectionKey = SectionKey)
+        internal static IServiceCollection ErniAcademyConfigureOptions<TOptions>(this IServiceCollection services, string sectionKey = SectionKey)
```

Comment:
Why this change? is TOptions a specific type of interface bounded to Erni Academy?

Reply:
done